### ISSUE TITLE
load pretrained model from lastest LOD

### ISSUE THREAD

**Issue by miaoYuanyuan** [podgorskiy/ALAE#59]
Dear @podgorskiy : 
    Thank you for your great work ALAE! when I Continue to train the LOD=7 , I load checkpoint from lod_6.pth, but the loss is 100 times larger than LOD6 , and the sample image is bad . not a normal RGB image. It seems not load checkpoint well.  When I sample images after each loss.backward ,I found the first picture(LOD=7, after the first loss.backward) is normal, as the  time goes, the picture is broken dowm. It seems checkpoint is load well, but somthine about LOD6 and LOD7 is lost, Counld you help me ? Thank you !

**Comment by miaoYuanyuan** [comment]
I know the reason, When load checkpoint ,we only load to model_s.encoder and model_s.decoder, but model not model_s is training and model_s is eval.
